tiledb/tests/test_array_from_cloud.py

```diff
@@ -0,0 +1,43 @@
+import tiledb
+import os
+import pytest
+
+tiledb_token = os.getenv('TILEDB_TOKEN')
+tiledb_path = os.getenv('TILEDB_PATH')
+s3_path = os.getenv('S3_PATH')
+
+@pytest.mark.skipif(tiledb_token is None, reason="No TileDB token found. Please set TILEDB_TOKEN env variable.")
```

Comment:
Let's also check `env["CI"]` and don't skip, because we don't want to skip this if we're running on CI in case there was a problem in the token setup -- then we would silently fail.